### ISSUE TITLE
IDEA Plugin - Tests are not connected with their source lines and log file prepends browser path to file path

### ISSUE THREAD

**Issue by GoogleCodeExporter** [manojjsm/js-test-driver#186]
```
What steps will reproduce the problem?
1. Run tests
2. Double-click on test result - WebStorm/IDEA should jump to that line
3. Double-click on test result in console - WebStorm/IDEA should jump to that 
line

What is the expected output? What do you see instead?

I'm really missing having the tests linked to their test source.

The log entries should have the server path stripped. As an example the 
following
http://localhost:9876/test/Z:/dgr/dev/js/crowdmashUber/dojoOopTesting/src-test/G
reeterTest.js:13
should be
Z:/dgr/dev/js/crowdmashUber/dojoOopTesting/src-test/GreeterTest.js:13

I'm not sure if that is enough to link the test with it's file, but if it's not 
it seems straightforward to parse the test result and update it appropriately.

Thanks!

What version of the product are you using? On what operating system?

JSTestDriver 1.2.2
JSTestDriver Plugin 1.2.2b
WebStorm WS-102.18


Please provide any additional information below.

```

Original issue reported on code.google.com by `coolr...@gmail.com` on 5 Jan 2011 at 5:01

**Comment by GoogleCodeExporter** [comment]
Original comment by `corbinrs...@gmail.com` on 5 Jan 2011 at 9:35

**Comment by GoogleCodeExporter** [comment]
```
Yes, this would be really great. Maybe hard to parse the message from the 
browser to figure out the file and line being referenced. Also I'm not sure yet 
how to link to a source location in IDEA.
```

Original comment by `aeagle22206` on 5 Jan 2011 at 11:02

**Comment by GoogleCodeExporter** [comment]
Original comment by `aeagle22206` on 5 Jan 2011 at 11:02
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Sergey.S...@gmail.com` on 11 May 2011 at 7:40
- Added labels: **IntelliJ**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v1.3.5.
```

Original comment by `Sergey.S...@gmail.com` on 20 Jun 2011 at 12:38
- Changed state: **Fixed**
